portfolio/static/js/validation.js

```diff
@@ -1,25 +1,13 @@
 $(function() {
   // Initialize form validation on the registration form.
   // It has the name attribute "registration"
-  $.validator.addMethod(
-        "regex",
-        function(value, element, regexp) {
-            var re = new RegExp(regexp);
-            return this.optional(element) || re.test(value);
-        },
-        "Please check your input."
-);
```

Comment:
Don't remove this

Reply:
Sorry to edit the validation.js containing regex method, Now I've just edited the validation.js from staticfiles and display.html with required condition. Anything else required.?

Reply:
nope, commit your changes so that I can review